modules/markdown/markdown_test.go

```diff
@@ -296,6 +296,7 @@ func TestRender_Commits(t *testing.T) {
 	test(sha, `<p><a href="`+commit+`" rel="nofollow">b6dd6210ea</a></p>`)
 	test(commit, `<p><a href="`+commit+`" rel="nofollow">b6dd6210ea</a></p>`)
 	test(tree, `<p><a href="`+src+`" rel="nofollow">b6dd6210ea/src</a></p>`)
+	test("commit "+sha, `<p>commit <a href="`+commit+`" rel="nofollow">b6dd6210ea</a></p>`)
```

Comment:
I won't ask for this PR but it would be better if the short sha is not hardcoded but derived from sha in test result (in the future ^^).